app/views/helpers/dateFields.scala.html

```diff
@@ -14,7 +14,11 @@
 @(form: Form[_], fieldName: String, id: Option[String] = None, isOptional: Boolean = false, hintMessageKey: String = "help.date", label: Option[String] = None)(implicit request: Request[_], messages: Messages)
 
 @dateErrors = @{
-    form.error(s"$fieldName").map(errors => ErrorMessage(content = Text(messages(errors.messages.head))))
+(form.error(s"$fieldName"), form.error(s"${fieldName}.day"), form.error(s"${fieldName}.month"), form.error(s"${fieldName}.year")) match
+            {
+                case (None, None, None, None ) => None
+                case _                   => Some(ErrorMessage(content = Text(messages("error.common.invalid.date"))))
+            }
 }
 
 <div id="@{id.getOrElse(fieldName)}">
```

Comment:
I think this can lead to duplicates (see: https://github.com/hmrc/business-rates-check-frontend/pull/742/files#diff-742289a9e03b48e3a5bc14f5eec6cd3312581da781deef7ff041006035c83284L20)

Maybe this to be consistent
```
<div id="@{id.getOrElse(s"${fieldName}-group")}">
```